swagger.yaml

```diff
@@ -1209,7 +1209,7 @@ definitions:
     required: ["id", "contacts", "description", "license", "links", "national_statistics", "title"]
     properties:
       canonical_topic:
-        description: "The canonical topic id for this dataset.  This indicates which topic this dataset belongs to within the website taxonomy."
+        description: "The canonical topic id for this dataset.  This indicates which topic this dataset belongs to within the website taxonomy. **Deprecated** this field is being deprecated and replaced by the "Themes" field."
```

Comment:
When I run this file through the swagger validator (https://editor.swagger.io/) there is a validation error about bad indentation at this line.